command-expansion-server/sql/commanding/tables/command_dictionary.sql

```diff
@@ -25,3 +26,17 @@ comment on column command_dictionary.version is e''
   'A human-meaningful version qualifier.';
 comment on constraint command_dictionary_natural_key on command_dictionary is e''
   'There an only be one command dictionary of a given version for a given mission.';
+
+create function trigger_update_at_timestamp()
```

Comment:
I think Dylan had a good naming scheme for these in his activity metadata PRs - I think this would be `trigger_command_dictionary_set_updated_at`. I think the key there is to include the table name, and the column name. (same goes for the name of the function)

Reply:
Good suggestion, makes it clearer on what is happening